go/runtime/registry/registry.go

```diff
@@ -386,38 +386,43 @@ func (r *runtime) run(ctx context.Context) {
 
 			// If this is a compute runtime and the active descriptor is not
 			// initialized, update the active descriptor.
-			if !activeInitialized && rt.Kind == registry.KindCompute {
+			if !activeInitialized && rt.IsCompute() {
 				if up := r.updateActiveDescriptor(ctx); up && !activeInitialized {
 					close(r.activeDescriptorCh)
 					activeInitialized = true
 				}
 			}
 
-			// Download bundles for the active and future versions.
-			now, err := r.consensus.Beacon().GetEpoch(ctx, consensus.HeightLatest)
-			if err != nil {
-				r.logger.Error("failed to get current epoch",
-					"err", err,
-				)
-				continue
-			}
-
-			var manifestHashes []hash.Hash
-			for i := len(rt.Deployments) - 1; i >= 0; i-- {
-				// Some deployments may lack a bundle manifest checksum,
-				// as it is optional.
-				if h := rt.Deployments[i].BundleChecksum; len(h) == hash.Size {
-					manifestHashes = append(manifestHashes, hash.Hash(h))
+			// Support hot-loading of runtime bundles only for compute nodes.
+			// Key managers must configure the bundle statically to prevent
+			// running different versions of the runtime.
+			if rt.IsCompute() {
```

Comment:
Maybe just `continue` if it's not compute and leave indentation as it was before?